PatientRegistrationApp/src/App.vue

```diff
@@ -90,7 +90,7 @@
 									<v-toolbar flat>
 										<v-toolbar-title class="text-wrap">Please ensure your information is correct</v-toolbar-title>
 									</v-toolbar>
-									<v-card flat><SinglePatientReviewSubmit/></v-card>
+									<v-card flat><SinglePatientReviewSubmit ref="singlepatientreviewsubmit"/></v-card>
```

Comment:
May you follow camelCase for the ref here and for HouseholdReviewSubmit (and update the references)?